server/db/migrations/patch-2-1.sql

```diff
@@ -0,0 +1,6 @@
+-- drop the visits table
+DROP TABLE visits;
+-- drop the user_pages table
+DROP TABLE user_pages;
+-- drop the users table
+DROP TABLE users;
```

Comment:
you guessed it. from [the docs](http://media.giphy.com/media/ryOujCIH2dyla/giphy.gif):

> DROP TABLE always removes any indexes, rules, triggers, and constraints that exist for the target table. However, to drop a table that is referenced by a view or a foreign-key constraint of another table, CASCADE must be specified.

Rather than rely on CASCADE, I just delete tables in reverse order of dependency, in hopes of making the migrations more easily scannable.